include/halo/lib/transforms/analyzer.h

```diff
@@ -65,12 +66,27 @@ class Analyzer final : public ModulePass {
     size_t size = 0;
   };
 
+  struct HW_Info {
```

Comment:
Use "HWInfo"? so it's more consistent with other type names like "TensorInfo"

Reply:
Sure, good catch! Updated.